.pylintrc

```diff
@@ -62,6 +62,7 @@ ignore-imports=yes
 #    too-many-statements
 # and fix those issues.
 
+disable=no-self-use,broad-except,too-many-statements,too-many-locals,too-many-branches,dangerous-default-value,not-context-manager,duplicate-code
```

Comment:
Ah, I missed this comment. Thanks.

* no-self-use: I don't think pulling those functions out of test-utils is a good choice. They will be used in many tests in future. Therefore I am in favor of using `@staticmethod`.
* too-many-locals, too-many-statements: will try to do that.